src/config.rs

```diff
@@ -1,24 +1,30 @@
 use serde::Deserialize;
 use std::{fs, path::Path};
 
-#[derive(Default, Debug, Deserialize, PartialEq)]
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(default)]
 pub struct Config {
     #[serde(default = "Config::default_apibase")]
```

Comment:
Is this needed after adding the `#[serde(default)]`?

Reply:
Yes, the annotation is the mark for using the type's implementation of `std::default::Default`. So, I fix the code as only using `#[serde(default)]` in toplevel. https://github.com/mackerelio-labs/mackerel-agent-rs/pull/20/commits/dafa2963ffc8dc70cbd851328ce88e786dfbb068